src/templates/earn-page/btc/index.tsx

```diff
@@ -1,22 +1,133 @@
 import { FC } from "react"
 import { graphql } from "gatsby"
-import { Box } from "@chakra-ui/react"
+import { Box, Stack, VStack } from "@chakra-ui/react"
 import RolePageTemplate from "../RolePageTemplate"
 import SectionTemplate from "../../home-page/SectionTemplate"
-import { H4 } from "../../../components"
+import { BodyLg, H3, H4, H5 } from "../../../components"
 import { LPCardGroup } from "../../../components/LPCard"
+import { TBTCStats } from "./TBTCStats"
+import { gql } from "graphql-request"
+import { TBTC_SUBGRAPH_URL } from "../../../config/subgraph"
+import useQuery from "../../../hooks/useQuery"
+import { LatestMint, LatestMints } from "./LatestMints"
+import ExternalButtonLink from "../../../components/Buttons/ExternalButtonLink"
+import { ExternalLinkHref } from "../../../components/Navbar/types"
+import { TBTCPartners } from "../../home-page/BTCRole/tBTCPartners"
+import { PageSection } from "../../../components/PageSection"
+import BlogPosts from "../../press-page/BlogPosts"
 
 const BTCPageTemplate: FC<any> = ({ data }) => {
-  const { btcInfo, interestedPools } = data.markdownRemark.frontmatter
+  const { btcInfo, tbtcPartners, interestedPools } =
+    data.markdownRemark.frontmatter
+
+  const { minters, guardians } = tbtcPartners
+
+  // TODO: The transactions don't contain the information about the user address,
+  // that did a reveal.
+  const {
+    isFetching,
+    data: tbtcTokenData,
+    error,
+  } = useQuery<{
+    tbtctoken: { currentTokenHolders: string; totalSupply: string }
+    transactions: LatestMint[]
+  }>(
+    TBTC_SUBGRAPH_URL,
+    gql`
+      query {
+        tbtctoken(id: "TBTCToken") {
+          currentTokenHolders
+          totalSupply
+        }
+        transactions(
+          where: { description: "Minting Finalized" }
+          first: 10
+          orderBy: timestamp
+          orderDirection: desc
+        ) {
+          amount
+          txHash
+          to
+          from
+          timestamp
+        }
+      }
```

Comment:
I've noticed that the `from`, `to` and `amount` aren't refer to the fields of `Transfer` event from tBTC v2 token(where the tokens were minted/transferred to a depositor) but actually to transaction logs of the `finalizeOptimisticMint` transaction- see https://github.com/threshold-network/tbtc-subgraph/blob/master/src/mappingTBTCVault.ts#L95-L97. So in that case the `from` is always a Minter address and `to` is `tBTCVault` contract address. And `amount` doesn't take into account fees.

I think the query should look like [this](https://api.thegraph.com/subgraphs/name/suntzu93/threshold-tbtc/graphql?query=%7B%0A++transactions%28%0A++++where%3A+%7Bdescription%3A+%22Minting+Finalized%22%7D%0A++++orderBy%3A+timestamp%0A++++orderDirection%3A+desc%0A++++first%3A+4%0A++%29+%7B%0A++++id%0A++++timestamp%0A++++deposits+%7B%0A++++++actualAmountReceived%0A++++++user+%7B%0A++++++++id%0A++++++%7D%0A++++%7D%0A++%7D%0A%7D):
```graphql
{
  transactions(
    where: {description: "Minting Finalized"}
    orderBy: timestamp
    orderDirection: desc
    first: 4
  ) {
    id // tx hash
    timestamp
    deposits {
      actualAmountReceived
      user {
        id // user eth address
      }
    }
  }
}
```

Reply:
7a4e83dded19035f4836890fcc188ab0df7db653